adafruit_sgp40.py

```diff
@@ -117,19 +155,74 @@ def _reset(self):
         try:
             self._read_word_from_command(delay_ms=50)
         except OSError:
-            # print("\tGot expected OSError from reset")
+            # Got expected OSError from reset
             pass
         sleep(1)
 
+    @staticmethod
+    def _temp_c_to_ticks(temperature):
```

Comment:
I prefer to be more explicit with method name, it could be confusing for folks, is this to convert Celsius to ticks or the temperature to ticks?

Reply:
So something along the lines of:

```suggestion
    def _celsius_to_ticks(temperature):
```

I had it as temperature because the default circuit python temperature is a type float with Celsius as the units. But explicit is the right way to go so if that's straight forward enough I'll make the adjustment

Reply:
Thanks that is pefect